File: JAVA_PROGRAMMING/ASSIGNMENTS/CONTROL_STATEMENTS/While_loop/program9.java

// WAP to print the reverse the given number.
//
class While{

	public static void main(String[] args){

		int num = 942111423;
		int rev= 0;
		while(num != 0){

			int rem = num % 10;
			rev = rev * 10 + rem;

			num /= 10;
		}

		System.out.println("Reverse is :  "+ rev);
	}
}

File: C_PROGRAMMING/C_PROGRAMS/Statements/2_practical/program9.c


//WAP to print the divisors of the given number by user
//
#include<stdio.h>
void main()
{
  int num;
  printf("Enter the Number:  ");
  scanf("%d",&num);

  for(int i=1;i<=num;i++)
  {
    if(num%i==0)
    {
      printf("\n%d ",i);
    }
  }
 printf("\n");
}
